oas

```diff
@@ -1,506 +1,354 @@
+            },
+            "SensorData": {
+                "title": "SensorData",
+                "type": "object",
+                "properties": {
+                    "timestamp": {
+                        "type": "string",
+                        "description": "Timestamp of the data point",
+                        "format": "date-time",
+                        "example": "2022-10-11T12:47:05.891905"
+                    },
+                    "measurements": {
+                        "$ref": "#/components/schemas/Measurements"
+                    }
+                },
+                "description": "Data Model for asset's sensor information"
             },
-            "example": [
-              "optional",
-              "additional",
-              "details"
-            ]
-          }
+            "TrackDataRequest": {
+                "title": "TrackDataRequest",
+                "required": [
+                    "GSIN",
+                    "SSCC"
+                ],
+                "type": "object",
+                "properties": {
+                    "GSIN": {
+                        "type": "string",
+                        "description": "GSIN of the returnable device. Required*",
+                        "example": "40212345678945787993"
+                    },
+                    "SSCC": {
+                        "type": "string",
+                        "description": "SSCC of the returnable asset. Required*",
+                        "example": "00512345678916851849"
+                    },
+                    "from": {
+                        "type": "string",
+                        "description": "The timestamp to start from. Optional",
+                        "format": "date-time",
+                        "example": "2022-10-10T12:47:05.893049"
+                    },
+                    "to": {
+                        "type": "string",
+                        "description": "The timestamp to end at. Optional",
+                        "format": "date-time",
+                        "example": "2022-10-11T12:47:05.893063"
+                    },
+                    "limit": {
+                        "type": "integer",
+                        "description": "Total number of data points to be returned. Optional",
+                        "example": 10
+                    }
+                },
+                "description": "Data Model for the data request, using GSIN and SSCC identifiers"
+            },
+            "TrackDataResponse": {
+                "title": "TrackDataResponse",
+                "type": "object",
+                "properties": {
+                    "identifier": {
+                        "type": "string",
+                        "description": "Physical identifier (MAC Address) of the returnable asset",
+                        "example": "12:34:56:78:90:AB"
+                    },
+                    "SSCC": {
```

Comment:
instead of copy/paste definitions for Identifier, SSCC, GSIN, define types ("component")

Reply:
You could define components that you refer to from the different object they are used (the same way you "reuse" SensorData)